hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/AlertsEngineImpl.java

```diff
@@ -372,9 +373,15 @@ private void reloadTrigger(Trigger trigger) {
         if (log.isDebugEnabled()) {
             log.debug("Reloading " + trigger);
         }
+
         // Look for the Trigger in the rules engine, if it is there then remove everything about it
         removeTrigger(trigger);
 
+        // If the trigger is not enabled then just stop after ensuring it's no longer loaded
```

Comment:
Ok, this validation can be removed.
It is not really necessary and it makes the cluster fails.

In a couple of lines below, there is protection for this:
```
if (trigger != null && trigger.isLoadable())
```
In distributed scenarios, the full trigger is not propagated, only ids, so the refresh to get the full data of the backend is necessary. 

Anyway, I have tested locally this PR without this validation and itest and cluster pass.